tests/telemetry_test.py

```diff
@@ -32,3 +32,73 @@ async def test_telemetry_hello(monkeypatch, optimizer: servo.configuration.Optim
 
     assert request.called
     assert expected in request.calls.last.request.content.decode()
+
+class TestDiagnosticStates:
+    def test_withhold_diagnostic_state(self) -> None:
+        state = servo.telemetry.DiagnosticStates("WITHHOLD")
+        assert state == servo.telemetry.DiagnosticStates.withhold
+
+    def test_send_diagnostic_state(self) -> None:
+        state = servo.telemetry.DiagnosticStates("SEND")
+        assert state == servo.telemetry.DiagnosticStates.send
+
+    def test_stop_diagnostic_state(self) -> None:
+        state = servo.telemetry.DiagnosticStates("STOP")
+        assert state == servo.telemetry.DiagnosticStates.stop
+
+@respx.mock
+async def test_diagnostics_request(monkeypatch, optimizer: servo.configuration.Optimizer) -> None:
+
+    # Simulate running as a k8s pod
+    monkeypatch.setenv("POD_NAMESPACE", "test-namespace")
+
+    # NOTE: Can't use servo_runner fixture; its init happens prior to setting of env var above
+    servo_runner = servo.runner.ServoRunner(servo.Servo(
+        config=servo.BaseServoConfiguration(name="archibald", optimizer=optimizer),
+        connectors=[], # Init empty servo
+    ))
```

Comment:
Don't need to simulate running in k8s for these tests

```suggestion
async def test_diagnostics_request(servo_runner: servo.runner.ServoRunner, optimizer: servo.configuration.Optimizer) -> None:
```